Makefile

```
@@ -45,3 +44,26 @@ tidy::
 	cd pkg/tests && go mod tidy
 	cd testing && go mod tidy
 	go mod tidy
+
+# Update the subtree in pkg/tf2pulumi/internal/tf with the latest code from terraform master
+TFPKGS=backend\|command\|provider-terraform\|provider-simple\|provider-simple-v6\|provisioner-local-exec\|cloud\|builtin\|grpcwrap\|initwd\|earlyconfig\|moduledeps\|plugin\|tfplugin5\|plugin6\|tfplugin6\|communicator\|providercache\|e2e\|legacy\|moduletest\|helper\|repl\|terminal
+pull_terraform::
+	git remote remove terraform || true
+	git remote add -f -t main --no-tags terraform https://github.com/hashicorp/terraform.git
+	git rm -rf ./pkg/tf2pulumi/internal/tf
+	git read-tree --prefix=pkg/tf2pulumi/internal/tf/ -u terraform/main:internal
+	# Remove the parts of Terraform we don't need
+	find ./pkg/tf2pulumi/internal/tf -mindepth 1 -maxdepth 1 -type d -regex ".*/tf/\($(TFPKGS)\)" -exec rm -rf {} \;
```

optional: `-exec` supports `+` instead of `;` to pass in as many paths as it can in one invocation.
So instead of one rm -rf per matched path, this could rm -rf them all in one invocation.
```suggestion
	find ./pkg/tf2pulumi/internal/tf -mindepth 1 -maxdepth 1 -type d -regex ".*/tf/\($(TFPKGS)\)" -exec rm -rf {} '+'
```